tasks/collect-data/collect-data.yaml

```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: collect-data
   labels:
-    app.kubernetes.io/version: "4.5.0"
+    app.kubernetes.io/version: "4.6.0"
```

Comment:
This is really more of a bugfix, so I would suggest 4.5.1 here.

Reply:
Updated, thanks